pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set opacity of current window to 90 percent:
+
+`picom-trans --current --opacity={{90}}`
+
+- Set opacity of window with name 'Firefox' to 90 percent:
+
+`picom-trans --name={{Firefox}} --opacity={{90}}`
+
+- Select window to set opacity for with mouse cursor:
+
+`picom-trans --select --opacity={{90}}`
+
+- Toggle opacity for window with name 'Firefox':
```

Comment:
```suggestion
- Toggle a specific window opacity:
```

Reply:
I'm not entirely sure, but I think `Firefox` refers to a window class here, not a single window. Is this correct @flipsi?

Reply:
No, as far as I understand, window name is different from window class and this matches the name.
At least I tested this and compared my query with the different values I got from `xprop`. 
Window name is what is most likely shown as a title in window decorations of window managers.